keyboards/handwired/yamsek/info.json

```diff
@@ -0,0 +1,73 @@
+                {"matrix":[0,4], "label":"P", "x":17, "y":1.375},
+                {"matrix":[0,5], "label":"-", "x":18, "y":1.75},
+                {"matrix":[1,0], "label":"H", "x":13, "y":2.25},
+                {"matrix":[1,1], "label":"J", "x":14, "y":2.125},
+                {"matrix":[1,2], "label":"K", "x":15, "y":2},
+                {"matrix":[1,3], "label":"L", "x":16, "y":2.125},
+                {"matrix":[1,4], "label":":", "x":17, "y":2.375},
+                {"matrix":[1,5], "label":"'", "x":18, "y":2.75},
+                {"matrix":[2,0], "label":"N", "x":13, "y":3.25},
+                {"matrix":[2,1], "label":"M", "x":14, "y":3.125},
+                {"matrix":[2,2], "label":"<", "x":15, "y":3},
+                {"matrix":[2,3], "label":">", "x":16, "y":3.125},
+                {"matrix":[2,4], "label":"?", "x":17, "y":3.375},
+                {"matrix":[2,5], "label":"=", "x":18, "y":3.75},
+                {"matrix":[3,0], "label":"Shift", "x":-3.5, "y":3.85},
+                {"matrix":[3,1], "label":"Enter", "x":-2.5, "y":4.3},
+                {"matrix":[3,2], "label":"Spac", "x":-1.45, "y":4.55},
+                {"matrix":[3,3], "label":"Del", "x":-0.25, "y":5.3},
+                {"matrix":[3,4], "label":"Mo(Arr)", "x":-0.25, "y":6.3},
+                {"matrix":[3,5], "label":"Mo(Fun)", "x":-1.25, "y":6.3},
+                {"matrix":[4,0], "label":"Esc", "x":0, "y":1.75},
+                {"matrix":[4,1], "label":"Q", "x":1, "y":1.375},
+                {"matrix":[4,2], "label":"W", "x":2, "y":1.125},
+                {"matrix":[4,3], "label":"E", "x":3, "y":1},
+                {"matrix":[4,4], "label":"R", "x":4, "y":1.125},
+                {"matrix":[4,5], "label":"T", "x":5, "y":1.25},
+                {"matrix":[5,0], "label":"Tab", "x":0, "y":2.75},
+                {"matrix":[5,1], "label":"A", "x":1, "y":2.375},
+                {"matrix":[5,2], "label":"S", "x":2, "y":2.125},
+                {"matrix":[5,3], "label":"D", "x":3, "y":2},
+                {"matrix":[5,4], "label":"F", "x":4, "y":2.125},
+                {"matrix":[5,5], "label":"G", "x":5, "y":2.25},
+                {"matrix":[6,0], "label":"LShift", "x":0, "y":3.75},
+                {"matrix":[6,1], "label":"Z", "x":1, "y":3.375},
+                {"matrix":[6,2], "label":"X", "x":2, "y":3.125},
+                {"matrix":[6,3], "label":"C", "x":3, "y":3},
+                {"matrix":[6,4], "label":"V", "x":4, "y":3.125},
+                {"matrix":[6,5], "label":"B", "x":5, "y":3.25},
+                {"matrix":[7,0], "label":"TG(Game)", "x":0, "y":6.25},
+                {"matrix":[7,1], "label":"Mo(NPad)", "x":-1.0, "y":6.25},
+                {"matrix":[7,2], "label":"Backs", "x":-1.0, "y":5.25},
+                {"matrix":[7,3], "label":"Alt", "x":0.2, "y":4.5},
+                {"matrix":[7,4], "label":"Ctrl", "x":1.25, "y":4.25},
+                {"matrix":[7,5], "label":"Shift", "x":2.25, "y":3.8}
```

Comment:
The order of entries here does not look correct — it must be the same as the order of arguments in the resulting layout macro (which would be generated from that data).  So it probably should start with:
```
                {"matrix":[4,0], "label":"Esc", "x":0, "y":1.75},
                {"matrix":[4,1], "label":"Q", "x":1, "y":1.375},
                {"matrix":[4,2], "label":"W", "x":2, "y":1.125},
                {"matrix":[4,3], "label":"E", "x":3, "y":1},
                {"matrix":[4,4], "label":"R", "x":4, "y":1.125},
                {"matrix":[4,5], "label":"T", "x":5, "y":1.25},
...
```
Also you have some negative coordinates here, which would probably cause the lint to complain; you need to change the coordinates so that the minimum values of `x` and `y` are 0.